docs/tutorials/sample-based-krylov-quantum-diagonalization.ipynb

```diff
@@ -0,0 +1,778 @@
+    "\n",
+    "Note that the Hamiltonian is a specific instance of the generic interaction-electron Hamiltonian,\n",
+    "\n",
+    "$$\n",
+    "\\begin{align*}\n",
+    "  H &= \\sum_{\\substack{p, q \\\\ \\sigma}} h_{pq} \\hat{a}^\\dagger_{p\\sigma} \\hat{a}_{q\\sigma}  +  \\sum_{\\substack{p, q, r, s \\\\ \\sigma \\tau}} \\frac{h_{pqrs}}{2} \\hat{a}^\\dagger_{p\\sigma} \\hat{a}^\\dagger_{q\\tau} \\hat{a}_{s\\tau} \\hat{a}_{r\\sigma} \\\\\n",
+    "  &= H_1 + H_2,\n",
+    "\\end{align*}\n",
+    "$$\n",
+    "\n",
+    "where $H_1$ consists of one-body terms, which are quadratic in the fermionic creation and annihilation operators, and $H_2$ consists of two-body terms, which are quartic. For the SIAM,\n",
+    "$$\n",
+    "H_2 = U \\hat{n}_{d\\uparrow}\\hat{n}_{d\\downarrow}\n",
+    "$$\n",
+    "\n",
+    "and $H_1$ contains the rest of the terms in the Hamiltonian. In order to represent the Hamiltonian programmatically, we store the matrix $h_{pq}$ and the tensor $h_{pqrs}$.\n",
+    "\n",
+    "### Position and momentum bases\n",
+    "\n",
+    "Due to the approximate translational symmetry in $H_\\textrm{bath}$, we don't expect the ground state to be sparse in the position basis (the orbital basis in which the Hamiltonian is specified above). The performance of SQD is guaranteed only if the ground state is sparse, that is, it has significant weight on only a small number of computational basis states. To improve the sparsity of the ground state, we perform the simulation in the orbital basis in which $H_\\textrm{bath}$ is diagonal. We call this basis the *momentum basis*. Because $H_\\textrm{bath}$ is a quadratic fermionic Hamiltonian, it can be efficiently diagonalized by an orbital rotation.\n",
+    "\n",
+    "### Approximate time evolution by the Hamiltonian\n",
+    "\n",
+    "To approximate time evolution by the Hamiltonian, we use a second order Trotter-Suzuki decomposition,\n",
+    "\n",
+    "$$\n",
+    "  e^{-i \\Delta t H} \\approx e^{-i\\frac{\\Delta t}{2} H_2} e^{-i\\Delta t H_1} e^{-i\\frac{\\Delta t}{2} H_2}.\n",
+    "$$\n",
+    "\n",
+    "Under the [Jordan-Wigner transformation](https://en.wikipedia.org/wiki/Jordan%E2%80%93Wigner_transformation), time evolution by $H_2$ amounts to a single [CPhase](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.CPhaseGate) gate between the spin-up and spin-down orbitals at the impurity site. Because $H_1$ is a quadratic fermionic Hamiltonian, time evolution by $H_1$ amounts to an orbital rotation.\n",
```

Comment:
```suggestion
    "Under the [Jordan-Wigner transformation](https://en.wikipedia.org/wiki/Jordan%E2%80%93Wigner_transformation), time evolution by $H_2$ amounts to a single [CPhase](docs/api/qiskit/qiskit.circuit.library.CPhaseGate) gate between the spin-up and spin-down orbitals at the impurity site. Because $H_1$ is a quadratic fermionic Hamiltonian, time evolution by $H_1$ amounts to an orbital rotation.\n",
```